lib/api/api-fetch.js

```diff
@@ -2,9 +2,10 @@
 
 'use strict'
 
-const Body = require('./readable')
 const Headers = require('./headers')
 const { kHeadersList } = require('../core/symbols')
+const { Readable } = require('stream')
+const { ReadableStream, TransformStream } = require('stream/web')
```

Comment:
How can we run these on Node < 16? Should we disable fetch?

Reply:
Yea, probably.

Reply:
Done